package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "api-server",
+  "version": "1.0.0",
+  "description": "An Express/Node.js based server designed to be a “model agnostic” REST API server, which can perform CRUD operations on any data model",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Beasley! This is causing any test that you have written not be able to run, could you switch it to the following please:
```
"test": "jest --coverage",
"test-watch": "jest --watchAll --coverage"
```